### ISSUE TITLE
There's no documentation

### ISSUE THREAD

**Issue by mungbeans2** [apple/live-caller-id-lookup-example#14]
There's a bit of documentation about how to build things but that's it, there's  no documentation giving a holistic overview of how to use it and what to do with it.

A couple of example questions:
- After building things then what? 
- As a service is being built for the live lookup, then how to run that service on a Mac?
- How to configure the live lookup extension in the app to use this? For example what values should be used for the LiveCallerIDLookupExtensionContext's serviceURL and tokenIssuerURL in order to be able to use the stuff in live-caller-id-lookup-example?

Basically documentation of what to actually do and how to use this stuff to get up and running. That doesn't exist.

Thank you

**Comment by mungbeans2** [comment]
I found the doc "Testing Live Caller ID Lookup", I guess that's an overview of what to do.

**Comment by karulont** [comment]
While we can always make documentation better, this specific issue: "Get Started documentation" is solved by https://swiftpackageindex.com/apple/live-caller-id-lookup-example/main/documentation/pirservice/testinginstructions.
